client/blocks/product-purchase-features/product-purchase-features-list/google-analytics-stats.jsx

```diff
@@ -14,8 +14,10 @@ export default localize( ( { selectedSite, translate } ) => {
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon="stats-alt"
-				title={ translate( 'Stats from Google Analytics' ) }
-				description={ translate( 'Connect to Google Analytics for the perfect complement to WordPress.com stats.' ) }
+				title={ translate( 'Connect to Google Analytics' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Connect Google Analytics' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/my-plan-google-analytics-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).